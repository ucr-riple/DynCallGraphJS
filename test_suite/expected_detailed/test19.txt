{
    'global (test19.js:11:1:11:11)': [
        'foo (test19.js:1:1:7:8)'
    ],
    'foo (test19.js:3:16:3:21)': [
        'bar (test19.js:8:1:10:8)'
    ],
    'bar (test19.js:9:16:9:23)': [
        'foo (test19.js:1:1:7:8)'
    ]
}