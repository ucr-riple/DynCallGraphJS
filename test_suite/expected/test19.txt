{ '(test19.js:11:1:11:11)':
   [ '(test19.js:1:1:7:8)' ],
  '(test19.js:3:16:3:21)':
   [ '(test19.js:8:1:10:8)' ],
  '(test19.js:9:16:9:23)':
   [ '(test19.js:1:1:7:8)' ] }